db/schema.rb

```diff
@@ -13,17 +13,20 @@
 
 ActiveRecord::Schema.define(version: 20160815140302) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
Is it OK to have this line in our schema.rb even when postgres is **not** being used?
And will this remain after migrating with a different db?